src/frontends/tensorflow/src/decoder_proto.cpp

```diff
@@ -194,10 +193,10 @@ ov::Any DecoderProto::get_attribute(const std::string& name) const {
         }
         FRONT_END_GENERAL_CHECK(pshape.is_static(), "Dynamic shapes are not supported for Tensor attribute.");
         const auto& tf_type = tensor_proto.dtype();
+        auto ov_type = get_ov_type(tf_type);
         FRONT_END_GENERAL_CHECK(
-            TYPE_MAP().count(tf_type),
+            ov_type != ov::element::undefined,
```

Comment:
May be `ov_type.is_static()`, otherwise, we can't create a tensor below.

Reply:
good catch, fixed